### PR TITLE
bus/nes: add support for a couple Chinese unlicensed carts

### PR DESCRIPTION
**Pull request by 0kmg**
Software list items promoted to working
---------------------------------------
Fang Kuai Zong Dong Yuan (Chi) (previously known as Dr. Mario II)
Korean Igo (Kor)
Ninja Ryukenden (Chi)

### CODE REVIEW

**Comment by 0kmg** [comment]
A bit of digging on the bootleg forums based on the origin of Dr Mario II dump leads to a cart photo and a few scant details: https://bootleggames.fandom.com/wiki/Fangkuai_Zongdongyuan. Need to update PR.

File: hash/nes.xml [hunk 1]
```diff
@@ -62224,6 +62208,31 @@ preliminary proto for the PAL version, still running on NTSC systems) or the gfx
 		</part>
 	</software>
 
+<!-- from cah4e3 -->
+	<software name="drmario2">
+		<description>Fang Kuai Zong Dong Yuan (Chi)</description>
```
Comment (cuavas) [inline hash/nes.xml:62213]: Does the published actually use “Chi” as an abbreviation for China?  If not, please don’t use it.  Use ISO-3166-1 Alpha-2 codes for country abbreviations if the manufacturer/publisher doesn’t have a consistent scheme.

Also, don’t use this style of spacing for Hanyu Pinyin.  Call it “Fāngkuài Zǒngdòngyuán” or something.

**Comment by etabeta78** [comment]
As I commented somewhere else (maybe in other commit), the choice of
3-letters non-standard abbreviations came from the following facts
- long names for cases like GBA (and in future DS), with up to 6 languages
in Euro, carts were not easily dealt with otherwise,
- ISO standard have shortcuts which are questionable if you think as an
English-native speaker (e.g. Ger for Germany and Spa for Spain are easier
then DE or ES).

If we want to go for a(ny) standard, please do not do this at this commit
level, but go through console lists and bulk replace all of them, or none.
Our softlists will become a bit more like TOSEC-named sets, but it would
still be much better that starting to use different standards in single
commits ;-)

   my 2c

**Comment by cuavas** [comment]
There’s enough rot already without adding more.  No new sets with the confusing non-standard abbreviations.  People are familiar with ISO abbreviations, they’re used everywhere.  Using non-standard abbreviations was always a bad idea.

**Comment by etabeta78** [comment]
Nobody objected when the choice was first presented in 2013. ;-)

My point was: let us bulk update all the lists, without forcing
inconsistencies in these spare commits.
However, since I am still a few weeks away from being able to contribute
actively again to the project, I think your point is more valuable

File: hash/nes.xml (hunk 1 above)
Comment (0kmg) [reply hash/nes.xml:62213]: Unabbreviated and suggested pinyin added.

**Comment by cuavas** [comment]
@0kmg could you please merge upstream or rebase so the the conflicts in nes_slot.h can be resolved?

**Comment by 0kmg** [comment]
> @0kmg could you please merge upstream or rebase so the the conflicts in nes_slot.h can be resolved?

Rebased. Looks okay now.

**Pull request merged**